HdArnoldMaterial fails to compile for the latest USD dev branch

Hey guys,

**Describe the bug**

Build Issue:

Trying to build against latest PXR USD dev branch and latest Arnold commit:

```
scons: Reading SConscript files ...
Building Arnold-USD:
 - Build mode: 'profile'
 - Host OS: 'linux'
 - Arnold version: '6.0.4.1'
00:00:00    77MB         | log started Tue Oct  6 10:04:58 2020
00:00:00    77MB         | Arnold 6.0.4.1 [e641c6ed] linux clang-9.0.1 oiio-2.2.1 osl-1.11.6 vdb-4.0.0 clm-1.1.1.118 rlm-12.4.2 optix-6.6.0 2020/09/25 14:25:06
00:00:00    77MB         | running on -----
00:00:00    77MB         |  1 x AMD Ryzen Threadripper 2990WX 32-Core Processor (32 cores, 64 logical) with 128686MB
00:00:00    77MB         |  NVIDIA driver version 440.82 (Optix 60600)
00:00:00    77MB         |  GPU 0: GeForce RTX 2080 SUPER @ 1860MHz (compute 7.5) with 7981MB (7510MB available) (NVLink:0)
00:00:00    77MB         |  CentOS Linux 7 (Core), Linux kernel 3.10.0-1127.10.1.el7.x86_64
00:00:00    77MB         |  soft limit for open files raised from 1024 to 4094
00:00:00    77MB         |  
00:00:00    77MB         | loading plugins from /home/sue.sauer/Documents/USDpython2.7/arnold_run/arnold_sdk/bin/../plugins ...
00:00:00    77MB         |  alembic_proc.so: alembic uses Arnold 6.0.4.1
00:00:00    79MB         |  usd_proc.so: usd uses Arnold 6.0.4.1
00:00:00    79MB         |  cryptomatte.so: cryptomatte uses Arnold 6.0.4.1
00:00:00    79MB         |  cryptomatte.so: cryptomatte_filter uses Arnold 6.0.4.1
00:00:00    79MB         |  cryptomatte.so: cryptomatte_manifest_driver uses Arnold 6.0.4.1
00:00:00    79MB         | loaded 5 plugins from 3 lib(s) in 0:00.00
00:00:00    80MB         |  
00:00:00    80MB         | releasing resources
00:00:00    80MB         |  unloading 3 plugins
00:00:00    80MB         |   closing alembic_proc.so ...
00:00:00    80MB         |   closing usd_proc.so ...
00:00:00    80MB         |   closing cryptomatte.so ...
00:00:00    79MB         |  unloading plugins done
00:00:00    79MB         | Arnold shutdown
scons: done reading SConscript files.
scons: Building targets ...
Install file: "LICENSE.md" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/LICENSE.md"
g++ -o build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/cmd/arnold_to_usd -Wl,--no-undefined -g -pg build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/cmd/main.o -L/home/sue.sauer/Documents/USDpython2.7/arnold_run/arnold_sdk/bin -L/home/sue.sauer/Documents/USDpython2.7/arnold_run/arnold_sdk/bin -L/home/sue.sauer/Documents/USDpython2.7/USD/lib -Lbuild/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/cmd -Lcmd -L/usr/lib -Lbuild/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/translator -Ltranslator -lai -lusd_translator -ltbb -lsdf -ltf -lusd -lar -lusdGeom -lusdShade -lvt -lusdLux -lgf -lusdVol -lusdSkel -lusdRender -lpython2.7 -lboost_python
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/cmd/arnold_to_usd" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/bin/arnold_to_usd"
Building SDK documentation with Doxygen 1.8.5 ...
file:///home/sue.sauer/Documents/USDpython2.7/USDARNOLD/arnold-usd/build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/docs/html
Install directory: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/docs/html" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/docs/html"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/arnold_usd.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/arnold_usd.h"
Install file: "ndr/api.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/ndr/api.h"
Install file: "ndr/discovery.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/ndr/discovery.h"
Install file: "ndr/ndrarnold.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/ndr/ndrarnold.h"
Install file: "ndr/parser.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/ndr/parser.h"
Install file: "ndr/utils.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/ndr/utils.h"
Install file: "render_delegate/api.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/api.h"
Install file: "render_delegate/basis_curves.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/basis_curves.h"
Install file: "render_delegate/config.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/config.h"
Install file: "render_delegate/constant_strings.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/constant_strings.h"
Install file: "render_delegate/debug_codes.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/debug_codes.h"
Install file: "render_delegate/hdarnold.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/hdarnold.h"
Install file: "render_delegate/instancer.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/instancer.h"
Install file: "render_delegate/light.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/light.h"
Install file: "render_delegate/material.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/material.h"
Install file: "render_delegate/mesh.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/mesh.h"
Install file: "render_delegate/openvdb_asset.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/openvdb_asset.h"
Install file: "render_delegate/points.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/points.h"
Install file: "render_delegate/render_buffer.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/render_buffer.h"
Install file: "render_delegate/render_delegate.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/render_delegate.h"
Install file: "render_delegate/render_param.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/render_param.h"
Install file: "render_delegate/render_pass.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/render_pass.h"
Install file: "render_delegate/renderer_plugin.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/renderer_plugin.h"
Install file: "render_delegate/shape.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/shape.h"
Install file: "render_delegate/utils.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/utils.h"
Install file: "render_delegate/volume.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/render_delegate/volume.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/api.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/api.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldAlembic.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldAlembic.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldBlackmanHarrisFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldBlackmanHarrisFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldBox.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldBox.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldBoxAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldBoxAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldBoxFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldBoxFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCamera.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCamera.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCameraAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCameraAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCatromFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCatromFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldClosestFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldClosestFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCollection.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCollection.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldColorManager.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldColorManager.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldColorManagerOcio.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldColorManagerOcio.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCone.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCone.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldConeAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldConeAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldContourFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldContourFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCryptomatteFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCryptomatteFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCryptomatteManifestDriver.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCryptomatteManifestDriver.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCurves.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCurves.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCurvesAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCurvesAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCylCamera.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCylCamera.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCylinder.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCylinder.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCylinderAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCylinderAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCylinderLight.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldCylinderLight.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDenoiseOptixFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDenoiseOptixFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDiffFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDiffFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDisable.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDisable.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDisk.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDisk.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDiskLight.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDiskLight.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDiskLightAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDiskLightAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDistantLight.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDistantLight.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDistantLightAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDistantLightAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriver.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDriver.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverDeepexr.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDriverDeepexr.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverExr.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDriverExr.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverJpeg.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDriverJpeg.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverNoOp.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDriverNoOp.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverPng.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDriverPng.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverTiff.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldDriverTiff.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldFarthestFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldFarthestFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldFisheyeCamera.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldFisheyeCamera.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldGaussianFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldGaussianFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldGinstance.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldGinstance.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldHeatmapFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldHeatmapFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldImplicit.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldImplicit.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldIncludeGraph.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldIncludeGraph.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldInstancer.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldInstancer.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldLight.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldLight.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldLightAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldLightAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldListAggregate.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldListAggregate.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldMaterialx.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldMaterialx.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldMerge.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldMerge.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldMeshLight.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldMeshLight.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldMeshLightAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldMeshLightAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldMitnetFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldMitnetFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldNurbs.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldNurbs.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldOperator.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldOperator.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldOptions.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldOptions.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldOrthoCamera.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldOrthoCamera.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPerspCamera.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldPerspCamera.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPerspCameraAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldPerspCameraAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPhotometricLight.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldPhotometricLight.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPlane.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldPlane.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPointLight.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldPointLight.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPoints.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldPoints.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPointsAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldPointsAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPolymesh.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldPolymesh.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPolymeshAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldPolymeshAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldProcedural.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldProcedural.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldProceduralCustom.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldProceduralCustom.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldQuadLight.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldQuadLight.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldQuadLightAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldQuadLightAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSetParameter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldSetParameter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSetTransform.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldSetTransform.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldShape.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldShape.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldShapeAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldShapeAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSincFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldSincFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSkydomeLight.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldSkydomeLight.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSkydomeLightAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldSkydomeLightAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSphere.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldSphere.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSphereAPI.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldSphereAPI.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSphericalCamera.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldSphericalCamera.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSpotLight.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldSpotLight.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldStringReplace.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldStringReplace.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSwitchOperator.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldSwitchOperator.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldTriangleFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldTriangleFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldUsd.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldUsd.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldUvCamera.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldUvCamera.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldVarianceFilter.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldVarianceFilter.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldVolume.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldVolume.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldVolumeImplicit.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldVolumeImplicit.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldVrCamera.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/arnoldVrCamera.h"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/tokens.h" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/include/schemas/tokens.h"
g++ -o build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/libusdArnold.so -Wl,--no-undefined -g -pg -shared build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/tokens.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldColorManager.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriver.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldOperator.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldShape.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldShapeAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCameraAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldOptions.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPerspCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPerspCameraAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldOrthoCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldFisheyeCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCylCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSphericalCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldVrCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldUvCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPointLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDistantLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDistantLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldQuadLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldQuadLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSpotLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSkydomeLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSkydomeLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCylinderLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDiskLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDiskLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldMeshLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldMeshLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPhotometricLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverJpeg.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverNoOp.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverPng.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverTiff.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverDeepexr.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDriverExr.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldBoxFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldTriangleFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCatromFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldMitnetFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldGaussianFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSincFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldBlackmanHarrisFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldClosestFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldFarthestFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldVarianceFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldHeatmapFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldContourFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDiffFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDenoiseOptixFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSphere.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSphereAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCylinder.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCylinderAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCone.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldConeAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDisk.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPlane.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPolymesh.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPolymeshAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldNurbs.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldBox.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldBoxAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCurves.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCurvesAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPoints.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldPointsAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldImplicit.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldGinstance.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldProcedural.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldListAggregate.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldVolume.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldVolumeImplicit.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldInstancer.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldColorManagerOcio.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldMaterialx.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldMerge.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSetParameter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldDisable.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSwitchOperator.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldSetTransform.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCollection.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldIncludeGraph.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldStringReplace.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldAlembic.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldUsd.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCryptomatteFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldCryptomatteManifestDriver.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/arnoldProceduralCustom.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/moduleDeps.os -L/home/sue.sauer/Documents/USDpython2.7/arnold_run/arnold_sdk/bin -L/home/sue.sauer/Documents/USDpython2.7/arnold_run/arnold_sdk/bin -L/home/sue.sauer/Documents/USDpython2.7/USD/lib -Lbuild/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas -Lschemas -L/usr/lib -L/usr/lib -L/home/sue.sauer/Documents/USDpython2.7/USD/lib -larch -lsdf -ltf -lusd -lar -lusdGeom -lusdShade -lvt -lusdLux -ltbb -lpython2.7 -lboost_python
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/libusdArnold.so" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/lib/libusdArnold.so"
Install file: "schemas/__init__.py" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/lib/python/UsdArnold/__init__.py"
g++ -o build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/_usdArnold.so -Wl,--no-undefined -g -pg -shared build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/wrapModule.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapTokens.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldColorManager.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDriver.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldOperator.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldShape.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldShapeAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCameraAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldOptions.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldPerspCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldPerspCameraAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldOrthoCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldFisheyeCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCylCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldSphericalCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldVrCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldUvCamera.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldPointLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDistantLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDistantLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldQuadLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldQuadLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldSpotLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldSkydomeLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldSkydomeLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCylinderLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDiskLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDiskLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldMeshLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldMeshLightAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldPhotometricLight.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDriverJpeg.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDriverNoOp.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDriverPng.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDriverTiff.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDriverDeepexr.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDriverExr.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldBoxFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldTriangleFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCatromFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldMitnetFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldGaussianFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldSincFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldBlackmanHarrisFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldClosestFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldFarthestFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldVarianceFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldHeatmapFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldContourFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDiffFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDenoiseOptixFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldSphere.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldSphereAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCylinder.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCylinderAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCone.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldConeAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDisk.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldPlane.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldPolymesh.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldPolymeshAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldNurbs.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldBox.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldBoxAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCurves.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCurvesAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldPoints.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldPointsAPI.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldImplicit.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldGinstance.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldProcedural.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldListAggregate.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldVolume.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldVolumeImplicit.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldInstancer.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldColorManagerOcio.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldMaterialx.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldMerge.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldSetParameter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldDisable.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldSwitchOperator.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldSetTransform.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCollection.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldIncludeGraph.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldStringReplace.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldAlembic.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldUsd.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCryptomatteFilter.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldCryptomatteManifestDriver.os build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/wrapArnoldProceduralCustom.os -L/home/sue.sauer/Documents/USDpython2.7/arnold_run/arnold_sdk/bin -L/home/sue.sauer/Documents/USDpython2.7/arnold_run/arnold_sdk/bin -L/home/sue.sauer/Documents/USDpython2.7/USD/lib -Lbuild/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas -Lschemas -L/usr/lib -L/usr/lib -L/home/sue.sauer/Documents/USDpython2.7/USD/lib -Lbuild/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas -Lschemas -larch -lsdf -ltf -lusd -lar -lusdGeom -lusdShade -lvt -lusdLux -ltbb -lpython2.7 -lboost_python -lusdArnold
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/_usdArnold.so" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/lib/python/UsdArnold/_usdArnold.so"
Install file: "plugInfo.json" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/lib/usd/plugInfo.json"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/generatedSchema.usda" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/lib/usd/usdArnold/resources/generatedSchema.usda"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/source/plugInfo.json" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/lib/usd/usdArnold/resources/plugInfo.json"
Install file: "build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/schemas/schema.usda" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/lib/usd/usdArnold/resources/usdArnold/schema.usda"
Install file: "render_delegate/plugInfo.json" as "/home/sue.sauer/Documents/USDpython2.7/Maya_arnold-usd/plugin/hdArnold/resources/plugInfo.json"
g++ -o build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/render_delegate/basis_curves.os -c -fPIC -fno-operator-names -std=c++14 -w -O3 -g -O0 -pg -fPIC -D_LINUX -DUSD_HAS_FULLSCREEN_SHADER -DHDARNOLD_EXPORTS -I/home/sue.sauer/Documents/USDpython2.7/arnold_run/arnold_sdk/include -I/home/sue.sauer/Documents/USDpython2.7/USD/include -I/usr/include -I/usr/include/python2.7 -Ibuild/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/render_delegate -Irender_delegate render_delegate/basis_curves.cpp
In file included from render_delegate/basis_curves.cpp:17:0:
render_delegate/material.h:77:10: error: ‘void pxrInternal_v0_20__pxrReserved__::HdArnoldMaterial::Reload()’ marked ‘override’, but does not override
     void Reload() override;
          ^~~~~~
scons: *** [build/linux_x86_64/gcc_profile/usd-0.20.11_arnold-6.0.4.1/render_delegate/basis_curves.os] Error 1
scons: building terminated because of errors.
```


**To Reproduce**
Steps to reproduce the behavior:
1. Install latest PXR usd and latest arnold USD try to build arnold

**Expected behavior**
For Arnold to compile


**Used Software Versions**
 - Arnold 6.0.4.1 [e641c6ed]
 - USD: https://github.com/PixarAnimationStudios/USD/commit/66f59e903f9b454b59ecdf514d3c61d59f5bde10
 - Compiler: [e.g. GCC 6.3.1]
 - OS:  CentOS Linux 7 (Core), Linux kernel 3.10.0-1127.10.1.el7.x86_64
 - Any 3rd-party app: Maya 2020.2

Thanks 
Sue

--- a/render_delegate/material.cpp
+++ b/render_delegate/material.cpp
@@ -465,8 +465,6 @@ void HdArnoldMaterial::Sync(HdSceneDelegate* sceneDelegate, HdRenderParam* rende
 
 HdDirtyBits HdArnoldMaterial::GetInitialDirtyBitsMask() const { return HdMaterial::DirtyResource; }
 
-void HdArnoldMaterial::Reload() {}
-
 AtNode* HdArnoldMaterial::GetSurfaceShader() const { return _surface; }
 
 AtNode* HdArnoldMaterial::GetDisplacementShader() const { return _displacement; }

--- a/render_delegate/material.h
+++ b/render_delegate/material.h
@@ -72,9 +72,11 @@ public:
 
     /// Reloads the shader.
     ///
+    /// Note: this function is a pure virtual in USD up to 20.08, but removed after.
+    ///
     /// Currently does nothing.
-    HDARNOLD_API
-    void Reload() override;
+   HDARNOLD_API
+    void Reload() { }
 
     /// Returns the Entry Point to the Surface Shader Network.
     ///

